src/wallet/rpcwallet.cpp

```diff
@@ -3030,6 +3030,10 @@ void FundTransaction(CWallet* const pwallet, CMutableTransaction& tx, CAmount& f
             },
             true, true);
 
+        if (options.exists("add_inputs") ) {
+            coinControl.m_add_inputs = options["add_inputs"].get_bool();
```

Comment:
Makes sense to allow `add_inputs = false` when no input is manually selected?

Reply:
It'll just fail with insufficient funds. I don't think it's necessary to special-case that.